include/dfrn.php

```diff
@@ -1811,6 +1813,7 @@ private static function process_relocation($xpath, $relocation, $importer) {
 
 		update_contact_avatar($relocate["avatar"], $importer["importer_uid"], $importer["id"], true);
 
+		/// @TODO decide to use dbm::is_result() here
```

Comment:
Unworkable `@TODO`, do it or don't, but since it's an UPDATE result, it's not crucial.